universal-application-tool-0.0.1/test/FunctionalTest.java

```diff
@@ -70,6 +70,17 @@ public void addPerson() {
     assertThat(contentAsString(result)).contains("person John with ID:", "added.");
   }
 
+  @Test
+  public void syncAddPerson() {
+    Http.RequestBuilder request =
+        fakeRequest(routes.PostgresController.syncAdd("John"))
+            .header(Http.HeaderNames.HOST, "localhost:" + testServerPort());
+    Result result = route(app, request);
+
+    assertThat(result.status()).isEqualTo(OK);
+    assertThat(contentAsString(result)).contains("person John with ID:", "synchronously added.");
```

Comment:
Assert that the database contains a new record.

Reply:
Done.